packages/manageiq-ansible-venv/Dockerfile

```diff
@@ -14,27 +14,25 @@ ENV RPM_SPEC=manageiq-ansible-venv.spec \
 # CentOS repos: rpm-build and copr-cli dependencies
 # EPEL: copr-cli
 RUN sed -i 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/subscription-manager.conf
-RUN dnf -y install \
+RUN dnf config-manager --setopt=tsflags=nodocs --save && \
+    dnf -y install \
       http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
       http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm \
       https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
-    dnf config-manager --setopt=tsflags=nodocs --save && \
     dnf -y install \
-      copr-cli \
+      make \
       gcc \
+      krb5-devel \
```

Comment:
Interesting, what required this?

Reply:
I matched this list to the one in the python_modules.ks.erb.  My plan was to try to take these out 1 by 1, but I wanted them to match 1 for 1 with what was there.